lib/dom.js

```diff
@@ -94,25 +94,39 @@ type MouseEventHandler = (event: MouseEvent) => mixed
 type MouseEventListener = {handleEvent: MouseEventHandler} | MouseEventHandler
 type KeyboardEventHandler = (event: KeyboardEvent) => mixed
 type KeyboardEventListener = {handleEvent: KeyboardEventHandler} | KeyboardEventHandler
+type TouchEventHandler = (event: TouchEvent) => mixed
+type TouchEventListener = {handleEvent: TouchEventHandler} | TouchEventHandler
+type WheelEventHandler = (event: WheelEvent) => mixed
+type WheelEventListener = {handleEvent: WheelEventHandler} | WheelEventHandler
 
 type MouseEventTypes = 'contextmenu' | 'mousedown' | 'mouseenter' | 'mouseleave' | 'mousemove' | 'mouseout' | 'mouseover' | 'mouseup' | 'click' | 'dblclick';
 type KeyboardEventTypes = 'keydown' | 'keyup' | 'keypress';
+type TouchEventTypes = 'touchstart' | 'touchmove' | 'touchend' | 'touchcancel';
+type WheelEventTypes = 'wheel';
```

Comment:
There are few more event types for WheelEvent triggers;
`type WheelEventTypes = 'wheel' | 'mousewheel' | 'DOMMouseScroll';`